packages/core/src/controllers/BlockchainApiController.ts

```diff
@@ -230,8 +230,8 @@ export const BlockchainApiController = {
       path: `/v1/convert/gas-price`,
       headers: {
         'Content-Type': 'application/json',
-        'x-sdk-type': sdkType || 'w3m',
-        'x-sdk-version': sdkVersion || 'html-wagmi-4.2.2'
```

Comment:
improvement: abstract some `getApiHeaders` method since they are all the same (or have the same base)